src/Playwright.CLI/Program.cs

```diff
@@ -0,0 +1,111 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Playwright.CLI
+{
+    class Program
+    {
+        private const string DriverEnvironmentPath = "PW_CLI_DRIVERPATH";
+
+        static void Main(string[] args)
+        {
+            string pwPath = GetFullPath();
+            var playwrightStartInfo = new ProcessStartInfo(pwPath, string.Join(' ', args))
+            {
+                UseShellExecute = false,
+                CreateNoWindow = true,
+                RedirectStandardError = true,
+                RedirectStandardInput = true,
+                RedirectStandardOutput = true
+            };
+
+            var pwProcess = new Process()
+            {
+                StartInfo = playwrightStartInfo,
+            };
+
+            playwrightStartInfo.EnvironmentVariables.Add("PW_CLI_TARGET_LANG", "csharp");
+            pwProcess.Start();
+
+            pwProcess.WaitForExit();
+            Console.WriteLine(pwProcess.StandardOutput.ReadToEnd());
+
+        }
+
+        private static string GetFullPath()
+        {
+            string envPath = Environment.GetEnvironmentVariable(DriverEnvironmentPath);
```

Comment:
Who sets this env variable?

Reply:
This would be the user calling the tool. If there's none set, we get a `null` and do _fallback_.